provider/pkg/provider/org_access_token.go

```diff
@@ -70,14 +76,17 @@ func (ot *PulumiServiceOrgAccessTokenResource) Diff(req *pulumirpc.DiffRequest)
 		}, nil
 	}
 
-	changes := pulumirpc.DiffResponse_DIFF_NONE
+	changes, replaces := pulumirpc.DiffResponse_DIFF_NONE, []string(nil)
 	if diffs.Changed("description") {
-		changes = pulumirpc.DiffResponse_DIFF_SOME
+		changes, replaces = pulumirpc.DiffResponse_DIFF_SOME, append(replaces, "description")
+	}
+	if diffs.Changed("admin") {
+		changes, replaces = pulumirpc.DiffResponse_DIFF_SOME, append(replaces, "admin")
```

Comment:
Should we update this to use `NewDetailedDiffFromObjectDiff` like [here](https://github.com/pulumi/pulumi-pulumiservice/blob/main/provider/pkg/provider/deployment_settings.go#L490)?

Reply:
Yes, but I'd suggest doing that in a followup.